SparkPost/2web2ui
# hotfix/SA-1715 - Fix remove button rendering in the Saved Reports Modal

[SA-1715](https://sparkpost.atlassian.net/browse/SA-1715)

### What Changed
- Fixed conditional rendering of removal button for active filter tags
- Moved active filters components to new file
- Fixed content rendering between filters within filter groupings

### How To Test
- Verify content renders correctly between filter groupings
- Verify remove button is no longer present on tags within the saved reports modal

### To Do
